test/qa-tests/jstests/libs/test_background_ops.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-global-assign: 0 no-native-reassign: 0 */
```

Comment:
The linter complains about the reassignment of a read-only global variable, `db`, on lines 245 and 251. I've disabled this linter check since this file is just copied from the `mongo` repo.